.github/workflows/ci.yml

```diff
@@ -12,10 +12,8 @@ jobs:
     strategy:
       matrix:
         include:
-          - scala-version: '2.13.6'
-            sbt-version: '0.13.18'
```

Comment:
The PR which updated Scala version to 2.13.6 had bad check - although on the LHS we could see that it's testing for scala version 2.13.6, it did tests for 2.10 and that's why they passed. Now I've got errors on CI saying that no project cross builds to 2.13.6. (And I think it's true since sbt is still using 2.12)